posthog/api/__init__.py

```diff
@@ -418,34 +418,54 @@ def register_grandfathered_environment_nested_viewset(
     projects_router.register(r"experiments", EnterpriseExperimentsViewSet, "project_experiments", ["project_id"])
     register_grandfathered_environment_nested_viewset(r"groups", GroupsViewSet, "environment_groups", ["team_id"])
     projects_router.register(r"groups_types", GroupsTypesViewSet, "project_groups_types", ["project_id"])
-    project_insights_router = projects_router.register(
-        r"insights", EnterpriseInsightsViewSet, "project_insights", ["project_id"]
+    environment_insights_router, legacy_project_insights_router = register_grandfathered_environment_nested_viewset(
+        r"insights", EnterpriseInsightsViewSet, "environment_insights", ["team_id"]
     )
     register_grandfathered_environment_nested_viewset(
         r"persons", EnterprisePersonViewSet, "environment_persons", ["team_id"]
     )
     router.register(r"person", LegacyEnterprisePersonViewSet, "persons")
 else:
-    project_insights_router = projects_router.register(r"insights", InsightViewSet, "project_insights", ["project_id"])
+    environemnt_insights_router, legacy_project_insights_router = register_grandfathered_environment_nested_viewset(
```

Comment:
environemnt misspelling

Reply:
Great catch! I think when typechecking runs `EE_AVAILABLE` is always true, which is why mypy never hits this. And in reality I think we also don't ship PostHog without `ee/` anymore, as `posthog-foss` is a bit unloved. But better not to break accidentally